.ci/wercker.sh

```diff
@@ -136,12 +137,9 @@ no-error-htmlunit)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  echo "checkouting project sources ..."
-  svn -q export https://svn.code.sf.net/p/htmlunit/code/trunk/htmlunit@r14923 htmlunit
-  cd htmlunit
- sed -i "s/            <version>2.28-SNAPSHOT/            <version>2.28-20171106.080245-12/" pom.xml
-  echo "Running checkstyle validation ..."
-  mvn -e compile checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
+  checkout_from https://github.com/HtmlUnit/htmlunit
+  cd .ci-temp/htmlunit
+  mvn -e compile -DskipTests=true -Dcheckstyle.version=${CS_POM_VERSION}
```

Comment:
This command is incomplete. We need more than a compile. Please see the pervious command and commands from other places. I don't see why actual command would change since they just moved from SVN to Git.

Reply:
Fixed.